tests/vmi_networking_test.go

```diff
@@ -457,4 +467,18 @@ var _ = Describe("Networking", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	Context("VirtualMachineInstance with custom PCI address", func() {
+		It("should configure custom Pci address", func() {
+			By("checking eth0 Pci address")
+			testVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.RegistryDiskFor(tests.RegistryDiskCirros), "#!/bin/bash\necho 'hello'\n")
```

Comment:
the `echo` is a custom command for the VM? Doesn't that mean that it will immediately exit?

Reply:
I don't think so. Look at other tests that we are running in this file.

Reply:
Yea, looks ok then.

Reply:
This command only needed for cloud-init, to avoid waiting for the cloud-init source on the VMI start(that can take more than 1 minute)